thirdparty/json/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@
 
 # suppress shadowing names warning
 set(JSON_SystemInclude ON CACHE BOOL "" FORCE)
+# disabled due to compiler issues
+set(CMAKE_INTERPROCEDURAL_OPTIMIZATION_RELEASE OFF)
```

Comment:
what issues are observed?

Reply:
![image](https://user-images.githubusercontent.com/21181928/146667385-620fe6e4-3ac8-4b9b-8a03-f071c48f1e99.png)
@ilyachur do we have a jira ticket for such issues?

Reply:
I suppose we need to compile json, offline_transformations and python API with LTO as a right solution.
But it's a separate task